package.json

```diff
@@ -79,7 +79,7 @@
     "react-redux": "^7.1.0",
     "react-redux-i18n": "^1.9.3",
     "react-router-dom": "^5.0.1",
-    "react-scripts": "3.0.1",
+    "react-scripts": "^3.3.0",
```

Comment:
Do we still use react-scripts?

Reply:
I don't think so.
But I wanted to avoid unexpected behaviour by deleting it in this PR (usage of dependencies of react-scripts)